### PR TITLE
EVG-16746 Disable execution tasks with display task

### PR DESCRIPTION
**Pull request by bynn**
[EVG-16746](https://jira.mongodb.org/browse/EVG-16746)

### Description 
execution tasks are now updated with display task

### Testing 
staging https://spruce-staging.corp.mongodb.com/version/638a617097b1d375e8784289/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

unit tests

### CODE REVIEW

File: model/task/task_test.go [hunk 1]
```diff
@@ -2178,24 +2178,25 @@ func TestDeactivateTasks(t *testing.T) {
 	require.NoError(t, db.ClearCollections(Collection, event.EventCollection))
 
 	tasks := []Task{
-		{Id: "t0"},
+		{Id: "t0", DisplayOnly: true, ExecutionTasks: []string{"t6"}},
```
Comment (hadjri) [inline model/task/task_test.go:2181]: Can we add a case where we check that a task that depends on the execution task `t6` gets deactivated?

**Review by hadjri — CHANGES_REQUESTED** [review]
Can we also add display task cases to `[TestTaskSetPriority](https://github.com/evergreen-ci/evergreen/blob/main/model/lifecycle_test.go#L29)`?

File: model/task/task_test.go (hunk 1 above)
Comment (bynn) [reply model/task/task_test.go:2181]: added

**Comment by bynn** [comment]
added tests for set task priority

**Review by hadjri — APPROVED** [review]

**Pull request merged**